airflow/ui/src/components/TrendCountChart.tsx

```diff
@@ -35,7 +35,7 @@ import { useColorMode } from "src/context/colorMode";
 
 ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Filler, Tooltip);
 
-export type ChartEvent = { timestamp: string };
+export type ChartEvent = { timestamp: string | null };
```

Comment:
This will work because in the following code everything will end up being `NaN` trying to compute things on the null value.

I think we should explicitely handle the null case, line 45.

```
    if (event.timestamp === null) {
      return;
    }
```

Event will be counted but not have time position because we don't know when the event appeared, which I think is correct:
![Screenshot 2025-02-12 at 11 14 11](https://github.com/user-attachments/assets/38b843f1-0969-454e-8863-d69863acc49d)

Reply:
think you may mean will **not** work?

Reply:
Added the additional check, does this look good?